premailer-rails.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency 'premailer', '~> 1.7', '>= 1.7.9'
-  s.add_dependency 'actionmailer', '>= 3', '< 6'
+  s.add_dependency 'actionmailer', '>= 3', '< 7'
```

Comment:
Why @fphilipe can we just remove the upper bound?

Removing the upper bound doesn't assert that the gem works with all versions, it just allow people to try themselves and even open PRs fixing what is broken. If you keep the upper bound we can't even try this gem with rails master and see what is broken.

Reply:
> Why @fphilipe can we just remove the upper bound?

Legit question. Why not?!

Just released [v1.10.3](https://rubygems.org/gems/premailer-rails/versions/1.10.3).